frontend/public/components/marketplace/kubernetes-marketplace-items.jsx

```diff
@@ -0,0 +1,222 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import * as PropTypes from 'prop-types';
+import {Link} from 'react-router-dom';
+import {CatalogTileView} from 'patternfly-react-extensions/dist/esm/components/CatalogTileView';
+import {CatalogTile} from 'patternfly-react-extensions/dist/esm/components/CatalogTile';
+import {FilterSidePanel} from 'patternfly-react-extensions/dist/esm/components/FilterSidePanel';
+import {EmptyState} from 'patternfly-react/dist/esm/components/EmptyState';
+import FormControl from 'patternfly-react/dist/esm/components/Form/FormControl';
+
+import {normalizeIconClass} from '../catalog-item-icon';
+
+export class MarketplaceTileViewPage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    const filters = {
+      byName: {
+        value: '',
+        active: false,
+      },
+      byProvider: {
+        redHat: {
```

Comment:
Can you make this a result of the operators that return from the API call instead of hard code?

Reply:
Good call, I'll see what I can do :)